frontend/src/components/pages/SearchReviews/SearchReviews.tsx

```diff
@@ -98,8 +99,19 @@ const SearchReviews = (): React.ReactElement => {
       minH="100vh"
     >
       <Center>
-        <Box w={["90%", "85%", "70%"]} py="10">
-          <SearchBox setSearchText={setSearchText} searchQuery={searchText} />
+        <Box w={["90%", "85%", "70%"]} pt="10">
+          <Box zIndex={100}>
+            <SearchBox setSearchText={setSearchText} searchQuery={searchText} />
+            <FilterBox
+              genreOptions={[{label: "Horror", value: 'horror'}, {label: "Fantasy", value: 'fantasy'}]}
```

Comment:
I'm not sure how I feel about hardcoding option and ages in the frontend, I think we should grab them from the `displayedReviews`. 

The way this would work is you loop over every review and you add it's genre to an array f it's not already in that array. 
For age options ordering matters, so 0-2 should show up before 3-5. For this, I think we should just grab the first number (i.e 3 for 3-5) and sort by that.